docs/tutorials/polybeam_simulation.ipynb

```diff
@@ -15,37 +15,26 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #31.    def perturbed_poly_beam(rotation):
</pre><p>Flagging for a future TODO, but I think this should be added to the tutorial data directory, and then we can just read it in.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HERA-Team/hera_sim/pull/238/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Made an issue: <a href="https://github.com/HERA-Team/hera_sim/issues/326" target="_blank">https://github.com/HERA-Team/hera_sim/issues/326</a></p><p> </p>

Reply:
<p>Great, thank you!</p>